R/distributions.R

```diff
@@ -334,3 +345,37 @@ sample_from_dist <- function(n, params){
     return(stats::runif(n = n, min = params$min, max = params$max))
   }
 }
+
+
+
+
+#' Plot a distribution
```

Comment:
should this function maybe be included in `plot.R` instead?

Reply:
Yes, indeed! moved with ee53e49e87eccebf5915a74cb1747b59b8a22420